localization/english/wc_schemes_l_english.yml

```
@@ -0,0 +1,51 @@
+﻿l_english:
+
+ stasis_trap_title:0 "Stasis Trap:"
+
+ start_stasis_trap:0 "Stasis Trap #weak (Scheme)#!"
+ stasis_trap_scheme:0 "Stasis Trap Scheme"
+ stasis_trap_scheme_desc:0 "[stun|E] your target"
```

Because it sounded too plain.
```suggestion
 stasis_trap_scheme_desc:0 "I will [stun|E] your [SCHEME.GetTarget.GetShortUINameNoTooltip]."
```